gui/src/node.vue

```diff
@@ -692,7 +692,7 @@
 import { locateServer, handleResponse } from "@/assets/js/utils";
 import CONST from "@/assets/js/const";
 import QRCode from "qrcode";
```

Comment:
Can we also replace this with `@nuintun/qrcode`?

Reply:
I think it is possible. Let me fix it.

Reply:
Done.

Reply:
I see the QR code is not center aligned, can you fix it?

Reply:
And it occupys more unnecessary width.
![image](https://user-images.githubusercontent.com/30586220/187065032-775b5c3f-a9a9-44c7-bb0b-8bd51693b5e0.png)

Reply:
Sometimes, it has incorrect ratio.
![image](https://user-images.githubusercontent.com/30586220/187065096-3095d25d-6767-44d4-91af-91a0f587fa7b.png)

Reply:
Sorry for lacking tests. I don't think it is appropriate to use @nuintun/qrcode to replace the current node-qrcode. Nuintun's one can only export a non-scalable SVG format qrcode image. I think I should revert the last commit and keep both libraryies.

Reply:
It seems img works fine. Code:

```diff
diff --git a/gui/src/node.vue b/gui/src/node.vue
index aff9e9b..a26b752 100644
--- a/gui/src/node.vue
+++ b/gui/src/node.vue
@@ -1436,7 +1436,7 @@ export default {
                         }</p>
                     </header>
                     <section class="modal-card-body lazy" style="text-align: center">
-                        <div><canvas id="canvas" class="qrcode"></canvas></div>
+                        <div><img id="qrcode" src=""/></div>
                         <div class="tags has-addons is-centered" style="position: relative">
                             <span class="tag is-rounded is-dark sharingAddressTag" style="position: relative" data-clipboard-text="${
                               res.data.data.sharingAddress
@@ -1474,20 +1474,15 @@ export default {
             if (row._type === CONST.SubscriptionType) {
               add = "sub://" + Base64.encode(add);
             }
-            const qrcode = new Encoder();
+            const enc = new Encoder();
 
-            qrcode.setEncodingHint(true);
-            qrcode.setErrorCorrectionLevel(ErrorCorrectionLevel.H);
-            qrcode.write(add);
-            qrcode.make();
+            enc.setEncodingHint(true);
+            enc.setErrorCorrectionLevel(ErrorCorrectionLevel.H);
+            enc.write(add);
+            enc.make();
 
-            const canvas = document.getElementById("canvas");
-            const context = canvas.getContext("2d");
-            const img = new Image();
-            img.onload = function() {
-              context.drawImage(img, 0, 0); // Or at whatever offset you like
-            };
-            img.src = qrcode.toDataURL();
+            const qrcode = document.getElementById("qrcode");
+            qrcode.src = enc.toDataURL(80 / Math.sqrt(add.length));
 
             let targets = document.querySelectorAll(".sharingAddressTag");
             let covers = document.querySelectorAll(".tag-cover");
@@ -1738,7 +1733,7 @@ table th {
   font-size: 40px;
 }
 
-.qrcode#canvas {
+#qrcode {
   min-height: 300px !important;
   min-width: 300px !important;
 }
```